Gemfile

```diff
@@ -28,5 +29,7 @@ group :development, :test do
   gem 'capybara'
   gem 'launchy'
   gem 'factory_girl_rails', '~> 4.5.0'
-  gem 'database_cleaner', '~> 1.4.0'
+  gem 'database_cleaner',   '~> 1.4.0'
```

Comment:
these were necessary for testing pdfs. You can see usage of them in model and controller.